packages/block-editor/src/store/reducer.js

```diff
@@ -1913,6 +1913,13 @@ export function blockEditingModes( state = new Map(), action ) {
 	return state;
 }
 
+export function openedBlockSettingsMenu( state = null, action ) {
+	if ( 'SET_OPENED_BLOCK_SETTINGS_MENU' === action.type ) {
+		return action?.clientId;
```

Comment:
For better precision, did we mean to return the default state of `null` instead of `undefined` in that scenario?

```suggestion
		return action?.clientId ?? null;
```

Reply:
Oh, good call, that helps give this reducer consistency with other reducers that track a client id or null 👍